sw/airborne/modules/computer_vision/viewvideo.c

```diff
@@ -187,6 +188,9 @@ static struct image_t *viewvideo_function(struct UdpSocket *viewvideo_socket, st
 #endif
   }
 
+  // Free all buffers
+  image_free(&img_jpeg);
+  image_free(&img_small);
```

Comment:
These should not be free'd. They are only free'd if the input image size has changed.